herddb-core/src/main/java/herddb/cluster/BookkeeperCommitLog.java

```diff
@@ -98,6 +98,7 @@
 
     private volatile boolean closed = false;
     private volatile boolean failed = false;
+    private final boolean skipTableConsistencyCheckDuringRecovery = true;
```

Comment:
Why do you need this flag?

Reply:
`                                    if (number.after(snapshotSequenceNumber) ) {
                                        if (LOGGER.isLoggable(Level.FINEST)) {
                                            LOGGER.log(Level.FINEST, "rec " + tableSpaceName + " #" + localEntryCount + " {0}, {1}",
                                                    new Object[]{number, statusEdit});
                                        }
                                        consumer.accept(number, statusEdit);
                                    } `

THe idea was to skip at this point if the consistency entry was received

Reply:
I think I can use entry.type